DesignSafe-CI/portal
# bug/DES-2834 - Navigating with "create new dataset" and "edit dataset"

## Overview: ##
After a user created or changed a dataset in the recon portal, they were sent back to the beginning of the workflow, rather than the previous menu so that they could add or edit more datasets. This was disorienting and inconvenient. 

## PR Status: ##


## Related Jira tickets: ##

* [DES-2834](https://tacc-main.atlassian.net/browse/DES-2834)

## Summary of Changes: ##
Changed the end of the functions that add or datasets to return the user to the dataset listing, rather than the home page for the recon portal admin

## Testing Steps: ##
1. Make sure your local user is part of the Rapid Admin groups in the CMS admin, so that they can access the rapid admin page
2. Go to designsafe.dev/recon-portal/admin and select the datasets of any event listed.
3. Add a new dataset. When the dataset is saved, you should be taken back to the dataset listing page. (see UI shot below)
4. Edit that same dataset. When the dataset is saved, you should be taken back to the dataset listing page. (see UI shot below)

## UI Photos:
![Screenshot 2024-06-21 at 9 52 27 AM](https://github.com/DesignSafe-CI/portal/assets/7462330/81f318bf-244e-4b34-91f4-805b20e85eb0)

## Notes: ##
